packages/block-library/src/site-tagline/block.json

```diff
@@ -8,6 +8,9 @@
 	"keywords": [ "description" ],
 	"textdomain": "default",
 	"attributes": {
+		"placeholder": {
```

Comment:
I'm not really sure we should a new block attribute just for preview purposes..

Reply:
I agree with @ntsekouras. Having attribute just for the block previews seems odd to me.

Let's follow the example of the [Site Title](https://github.com/WordPress/gutenberg/pull/35590) and [Site Logo](https://github.com/WordPress/gutenberg/pull/35588) blocks.

Reply:
Thanks for the feedback @ntsekouras and @Mamaduka! Here's how it would look reverted:

<img src="https://user-images.githubusercontent.com/14988353/221042447-22b1fed9-483d-4159-b604-0d0b033d2341.png" />

Just to share my thinking about introducing it: I don't feel too strongly about it, but my main concern with not having a placeholder value is that it could be fairly common for folks to not have a site tagline, but very uncommon to not have a site title, so there's a higher likelihood that the `Write site tagline...` text will display in the preview within the Style Book. If a user goes to click that, then they'll be taken to global styles, and they're not able to edit it. We already have the `placeholder` attribute and similar behaviour on the button, heading, image, list, list item, and paragraph blocks, so I thought it'd be fairly innocuous.

That said, I very much agree that it's good not to have extra attributes if they're not needed, and by reverting there's nothing stopping us from tweaking how previews work in future follow-ups, so I'm happy for us to proceed with reverting. I have a PR open for that over in https://github.com/WordPress/gutenberg/pull/48383.

Reply:
I agree UX might be confusing with the default placeholder, but maybe this is something we need to resolve separately. Maybe change a text copy.

Fun fact: WP only removed the default tagline [a few months ago](https://core.trac.wordpress.org/changeset/53815), and it broke our E2E tests 😄

Reply:
> but maybe this is something we need to resolve separately. Maybe change a text copy.

I agree — looking over this again today, I wasn't 100% sold on the placeholder approach as the preview should really be previewing a finished / proper Site Tagline rather than the placeholder state since the placeholder has reduced opacity.

> Fun fact: WP only removed the default tagline [a few months ago](https://core.trac.wordpress.org/changeset/53815), and it broke our E2E tests 😄

Whoa! I didn't know that, good to know 😄

Reply:
I agree on just using the placeholder the block provides if the site doesn't have a site tagline set.

Reply:
Thanks for confirming!